crates/bevy_input/src/axis.rs

```diff
@@ -22,7 +28,11 @@ where
     T: Copy + Eq + Hash,
 {
     pub fn set(&mut self, axis: T, value: f32) -> Option<f32> {
-        self.axis_data.insert(axis, value)
+        if value < self.min || value > self.max {
```

Comment:
I'm unsure if this is the right behavior, or if we should clamp the values instead. I think I lean towards the latter, but want more opinions.